integration/update_array_compat_test.go

```diff
@@ -238,3 +238,60 @@ func TestUpdateArrayCompatPullAll(t *testing.T) {
 
 	testUpdateCompat(t, testCases)
 }
+
+func TestUpdateArrayCompatAddToSetEach(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]updateCompatTestCase{
+		"Document": {
+			update: bson.D{{"$addToSet", bson.D{{"v", bson.D{
+				{"$each", bson.A{bson.D{{"field", int32(42)}}}},
+			}}}}},
+			skipForTigris: "Mixed types are not supported for Tigris.",
```

Comment:
So we skip all the tests for Tigris? Don't data sets like `ArrayInt32s` fit with a bit different query -- smth like
`{"$each", bson.A{int32(42), jnt32(43)}}`? I like the idea of the test with `field`, but I think we can have at least one case that would fit Tigris too.

Reply:
If we want to test only `Int32` we should filter by `array-int32s` then, because there are arrays with doubles which would produce different result from MongoDB (Tigris would convert int32 to float). If we ok with that, then I can apply some filters.